sentry-samples/sentry-samples-android/src/main/AndroidManifest.xml

```diff
@@ -109,7 +113,7 @@
         <meta-data android:name="io.sentry.traces.activity.auto-finish.enable" android:value="false" />
 
         <!-- how to enable the UI auto instrumentation for tracing -->
-        <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />
+<!--        <meta-data android:name="io.sentry.traces.user-interaction.enable" android:value="true" />-->
```

Comment:
I understand the problem, but disabling another feature isn't the best solution.
Maybe we create a new sample app for profiling as suggested before?